docs/canarynet/canarywalletguide.md

```diff
@@ -0,0 +1,69 @@
+# **Canary Net Guide**
+
+## This is a guide on how to get onto the Automata Canary Net for Chrome users(recommended).
+
+1. Go to website url - ‘Dashboard.ata.network‘
+   *You should reach a website which looks like the picture below.* ![img](https://lh4.googleusercontent.com/AfyfULXVPqAnCPDTxqql1RIly8zR4y36EBpFhRtpyuby8rhnNlBMuGoZn3OtoHV4TQLDaUY1U-mkZ0tr4jeoPZH7hkfOHhbe3UNB-h1_6Fsl4638eg2-9PfZrGw5dlarcCqjD4pH=s0)
+
+2. Hover over the Accounts taskbar and click on Accounts.
+   
+   *Picture showing what is needed to be clicked.*
+   
+   ![img](https://lh5.googleusercontent.com/rBK0MBvG20Vny58OR1mlHL-tathc6jKwWk4XAmkF1Bpxu3FDKOKWFZP452AJtwDx5bONQdaYc5Lx54w_risIeVYTNCGMTCJ1860WgbclfHXuoIxAzIqWKGe1Y4qRDdXv6PGxSOrW=s0)
```

Comment:
Please upload the image to the repository, and refer it as an internal link

Reply:
uploaded all image to assets/canaryimg